### ISSUE TITLE
Great Job!!

### ISSUE THREAD

**Issue by chaosoft** [winunix/system-config-samba-wx#1]
First off!! I love that you ported this to 20.04 Thanks

It installs just fine, but when going to lunch this it waits; then just closes. 

In the past just adding

sudo touch /etc/libuser.conf

Fixes that issue, but it didn't 
Any ideas would be appreciated 
Chris

**Comment by ailtonbsj** [comment]
Hi,
Thanks for your question.

What system are you using actuallty?
I tested on Lubuntu 18.04 LTS and Lubuntu 20.04 LTS and works well.
If you run `pkexec system-config-samba`on terminal whats message appears?

**Comment by chaosoft** [comment]
You're right!!  I should have mentioned what I'm using... I'm currently on Kubuntu 20.04… 
Also, after I uninstalled it, Then reinstalled the package it works LOL. I want to thank you again for hard work porting this over.

**Comment by ailtonbsj** [comment]
> Also, after I uninstalled it, Then reinstalled the package it works LOL.

Glad it worked for you. 👏

I installed Kubuntu 20.04 on a virtual machine and installed the app using winunix respository:

```bash
wget -qO - "https://winunix.github.io/debian/public.key" | sudo apt-key add -
echo "deb https://winunix.github.io/debian focal main" | sudo tee /etc/apt/sources.list.d/winunix-focal.list
sudo apt update
sudo apt install system-config-samba-focal
```

Everything's work well and I could to share the folders.

I think that if you are using Kubuntu you don't need this app because `Dolphin` has the feature to share folders just selecting one folder e going to `Properties  > Share`.

But even so thanks for test this code! 🤝

**Comment by chaosoft** [comment]
I agree that Kubuntu uses dolphin with shares, but I have used this tool for as long as I can remember. It is a clean app I think I have used it during my Debian days… Switched to Lubuntu Then Kubuntu this past year when I updated to a Ryzen chip. I tend to use my systems for 10 yrs or more, so I like longevity... The tool is so easy to use, I used to just do it by command line in Knopix days, and Red hat  before it went paid. So thanks again for the work, my long-winded response, I will be sure to pass this on to anyone who needs this tool...
